docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -4,5 +4,152 @@
 Dietary Iron Deficiency (Iron Deficiency Anemia)
 ================================================
 
+Disease Description
+-------------------
+
+Generally, **anemia** is a condition defined by a deficiency of red blood cells 
+or a deficiency of hemoglobin in the blood. Anemia is typically classified by 
+hemoglobin concentrations below a defined threshold that varies by age and sex. 
+Severity of anemia is similarly classified according to ranges of hemoglobin 
+concentrations. Anemia is associated with increased morbidity and mortality and 
+symptoms of anemia often include weakness, fatigue, and difficulty 
+concentrating.
+
+ Notably, anemia may be caused by many diverse factors. Examples of factors 
+ that may cause anemia include genetic mutations in hemoglobin genes, acute or 
+ chronic blood loss, altered red blood cell morphology, inadequate nutritional 
+ intake, and others.
+
+**Iron deficiency anemia** is a type of anemia that is due to insufficient 
+iron levels, which lead to a deficiency of hemoglobin in the blood. Notably, 
+iron deficiency anemia can occur when dietary intake of iron is insufficient, 
+although it may occur in other situations as well, such as when iron is lost 
+through bleeding (ex: menstrual disorders, hookworm disease, etc.). Iron 
+deficiency anemia is the most common cause of anemia globally in most 
+populations.
```

Comment:
This sounds a bit redundant---by 'populations' do you mean something along the lines of most age-sex (-race?) groups?